cfgov/cfgov/settings/base.py

```diff
@@ -677,14 +677,18 @@
 
 # Email popup configuration. See v1.templatetags.email_popup.
 EMAIL_POPUP_URLS = {
-    'debt': [
-        '/ask-cfpb/what-is-a-statute-of-limitations-on-a-debt-en-1389/',
-        '/ask-cfpb/what-is-the-best-way-to-negotiate-a-settlement-with-a-debt-collector-en-1447/',
-        '/ask-cfpb/what-should-i-do-when-a-debt-collector-contacts-me-en-1695/',
-        '/consumer-tools/debt-collection/',
-    ],
-    'oah': [
-        '/owning-a-home/',
-        '/owning-a-home/mortgage-estimate/',
-    ],
+    'debt': {
```

Comment:
Did you consider adding the language setting to the popup-specific config in the template files [here](https://github.com/cfpb/cfgov-refresh/tree/master/cfgov/jinja2/v1/_includes/organisms/email-popup)? For example there could be a new `popup_language` setting defined in the template [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/organisms/email-popup/oah.html).

This works well but it feels a little bit complicated to have the popup URLs grouped under language like this, since language is just one configuration option that we want to pass in. If there's some additional option in future that we also need to pass, it might get difficult to do that here.

Reply:
I like that much better. I didn't know it was possible. Fixed in https://github.com/cfpb/cfgov-refresh/pull/3922/commits/876592dab427f5ba0f56a76eee8a75be119f43ca